brage-import/src/main/java/no/sikt/nva/brage/migration/lambda/PublicationComparator.java

```diff
@@ -97,15 +100,21 @@ private static boolean atleasOneContributorMatch(Publication incomingPublication
 
     private static Stream<String> containsContributor(List<Contributor> incomingContributors,
                                                       Contributor contributor) {
-        var name = contributor.getIdentity().getName();
-        return incomingContributors.stream()
-                   .map(Contributor::getIdentity)
+        var lastName = getLastName(contributor);
+        return isNull(lastName)
+                   ? Stream.empty()
+                   : incomingContributors.stream()
+                         .map(PublicationComparator::getLastName)
+                         .filter(Objects::nonNull)
+                         .filter(lastName::equals);
+    }
+
+    private static String getLastName(Contributor contributor) {
+        return Optional.ofNullable(contributor.getIdentity())
                    .map(Identity::getName)
-                   .filter(value -> levenshteinDistanceIsLessThan(
-                       value,
-                       name,
-                       MAX_ACCEPTABLE_CONTRIBUTOR_NAME_LEVENSHTEIN_DISTANCE)
-                   );
+                   .map(name -> name.split(StringUtils.SPACE))
+                   .map(nameArray -> nameArray[nameArray.length - 1])
+                   .orElse(null);
```

Comment:
Not sure what these methods are doing here at all

Reply:
It gets last name of contributor. Splitting "name" by whitespaces and getting last value.